packages/ui-docs-client/src/App/index.js

```diff
@@ -102,17 +102,35 @@ class App extends Component {
     this.state = {
       showMenu: showTrayOnPageLoad,
       themeKey: Object.keys(props.themes)[0],
-      layout: 'large'
+      layout: 'large',
+      versionsData: {
+        latestVersion: 'v8',
+        previousVersions: ['v7', 'v6']
```

Comment:
I'm not sure about this. I know the fetch will update the data, but probably confusing to hardcode v8 here too. I would set the initial state to null or {}, and handle that in the code (like not rendering the version select until we have the version data, just the current version heading as before)